CIP-0052/README.md

```diff
@@ -222,6 +222,7 @@ DApps are used by *DApp users*, and built by *DApp developers*. Audit is perform
 | Hachi           | https://hachi.one                       | team@hachi.one        |
 | MLabs           | https://mlabs.city                      | info@mlabs.city      |
 | Runtime Verification           | https://www.runtimeverification.com | contact@runtimeverification.com  |
+| Tweag                | https://www.tweag.io/                  | sales@tweag.io                  |
```

Comment:
small tiny thing but I would avoid / at the end of the url apart from this LGTM

https://www.tweag.io/  -> https://www.tweag.io/